controller/lib/core/sensors.cpp

```diff
@@ -107,3 +141,15 @@ float get_pressure_reading(AnalogPinId pinId) {
   // Sensitivity of all pressure sensors is 1 V/kPa; no division needed.
   return ((float)runningSum * ADC_LSB);
 }
+
+float convert_diff_pressure_to_volumetric_flow(
```

Comment:
Note to self, I have not yet reviewed the math here, want to wait for things to be rearranged a bit.